todo_app/database.py

```diff
@@ -0,0 +1,16 @@
+import os
+import certifi
+import pymongo
+import app
+from pymongo import MongoClient
+
+dbpassword = os.environ['dbpassword']
+
+CONNECTION = f'mongodb+srv://marklawrence:{dbpassword}@cluster0.ukgck.mongodb.net/myFirstDatabase?retryWrites=true&w=majority&ssl=true'
+MONGODB = "todo_list"
+MONGOCOLLECTION = "todo_items"
+
+def mongodb_connection():
+    mongo_client = MongoClient(CONNECTION, tlsCAfile=certifi.where())
+    dbname = mongo_client[MONGODB]
+    return dbname
```

Comment:
As far as I can tell, you always access this db then immediately access the collection - so returning `dbname[MONGOCOLLECTION]` directly might save you a few lines everywhere?